tuplex/core/src/DataSet.cc

```diff
@@ -38,21 +38,17 @@ namespace tuplex {
     }
 
     std::shared_ptr<ResultSet> DataSet::collect(std::ostream &os) {
-        return take(-1, os);
+        return take(std::numeric_limits<size_t>::max(), 0, os);
```

Comment:
shouldn't this be max(), max()? pls. explain in doxygen the special values for the params

Reply:
Added some docs in take. Collect is define as having either top or bottom limit as size_t::max() (or both), so this is fine